media/js/wiki-compat-tables.js

```diff
@@ -1,5 +1,30 @@
 (function(win, doc, $) {
 
+    var $compatWrapper = $('.bc-api');
+    // check there's a table with at least one row
+    if($compatWrapper.length) {
+        $compatWrapper.each(function() {
+            var $thisWrapper = $(this);
+            if($thisWrapper.find('tbody tr').length) {
+                $thisWrapper.removeClass('hidden');
+            }
+        });
+    } else {
+        return;
```

Comment:
@darkwing This is supposed to stop the function defined on line 1 from executing? Right?